packages/fastify-siwe/src/api.test.ts

```diff
@@ -58,6 +73,31 @@ describe('signInWithEthereum', () => {
     expect(authResponse.message).to.deep.equal(message)
```

Comment:
Can we return and assert some more info - like which address am I authenticated with?